api/app/Models/PoolCandidate.php

```diff
@@ -250,9 +252,20 @@ public static function scopeExpiryStatus(Builder $query, ?string $expiryStatus)
         return $query;
     }
 
-    public function scopeNotDraft(Builder $query): Builder
+    public function scopeNotDraft(Builder $query, $where, bool $notDraft = true): Builder
     {
-        return $query->whereNotIn('pool_candidate_status', ['DRAFT', 'DRAFT_EXPIRED']);
+        Log::debug([
+            'notDraft' => $notDraft,
+            'where' => $where,
+        ]);
+        $user = Auth::user();
+        $canViewAny = $user && $user->isAbleTo("view-any-submittedApplication");
```

Comment:
Why are you checking the permission here?

Reply:
You know, I have no clue why I put this here 🤔 

Removed in [remove debug/permission check in application draft scope](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5815/commits/81c9a6f5527a011a8775eb7bd4d0b0dde65e23bc)